### PR TITLE
RA: Multi locale urls

### PR DESCRIPTION
**Pull request by arothuis**
[131975453](https://www.pivotaltracker.com/story/show/131975453)

This adds global view parameters, allowing multi-locale parameters to be injected in the view.
A multi-locale manual URL is configured in this PR.

#### Related
SURFnet/Stepup-SelfService#112

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -68,6 +68,9 @@
             },
             {
                 "file": "app/config/samlstepupproviders_parameters.yml"
+            },
+            {
+                "file": "app/config/global_view_parameters.yml"
```
Comment (rjkip) [inline composer.json]: How does incenteev handle 'deep parameters", ie. parameters that are objects (ie. `manual_url`)? IIRC one has to answer with JSON or something.

File: src/Surfnet/StepupRa/RaBundle/Exception/AssertionFailedException.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+use Assert\AssertionFailedException as AssertAssertionFailedException;
+
+class AssertionFailedException extends InvalidArgumentException implements AssertAssertionFailedException
+{
+    private $propertyPath;
+    private $value;
+    private $constraints;
+
+    // @codingStandardsIgnoreStart Compliance with beberlei/assert's invalid argument exception
```
Comment (rjkip) [inline src/Surfnet/StepupRa/RaBundle/Exception/AssertionFailedException.php]: You can leave it, but there's also `\Assert\Assertion::createException()` you could overwrite :smiley:

File: composer.json (hunk 1 above)
Comment (arothuis) [inline composer.json]: Yes. This is communicated to the user/developer by showing the default value in the same representation:
```
Some parameters are missing. Please provide them.
support_url ({ en_GB: 'https://support.surfconext.nl/faq-strong-authentication', nl_NL: 'https://support.surfconext.nl/faq-sterke-authenticatie' }): 
```

File: src/Surfnet/StepupRa/RaBundle/Exception/AssertionFailedException.php (hunk 2 above)
Comment (arothuis) [inline src/Surfnet/StepupRa/RaBundle/Exception/AssertionFailedException.php]: Ah, I did not know that. I kept it in line with how we did things in Profile last year.

File: composer.json (hunk 1 above)
Comment (rjkip) [inline composer.json]: Meh. It's as best as it gets, though, I guess :)

**Pull request merged**